lib/orgmode.js

```diff
@@ -272,6 +273,26 @@ class Orgmode extends OutlinesArrayList {
     }, {});
   }
 
+  ///**
+  // * @property {Object} overview - the overview of this document
+  // */
+  //get blocks() {
+  //  return this._ast.options.reduce((map, item) => {
+  //    map[item.name.toLowerCase()] = item.value;
+  //    return map;
+  //  }, {});
+  //}
+
+  /**
+   * @method findBlockByName(name)
+   * @param {Number} name
+   * @return {Block} return the blocks
+   */
+  findBlockByName(name) {
+    return this._ast.blocks.filter(
+      (block) => block.name === name
+    );
+  }
```

Comment:
> The class `OutlinesArrayList` don't have field `_ast`

This is not exact right in our example, here we wouldn't have any instance of `OutlinesArrayList`, it's more like the `interface` from other PL, so we should just assume that the `this._ast` is in the instance object.

BTW, because actually block is differ from the outline in orgmode, that's why I asked the relationship between them at [this comment](https://github.com/weflex/node-orgmode/pull/9#discussion_r67652744), and my thought is to create a new interface/class which do attach the block-related functions inside the `Orgmode` object.